### PR TITLE
Bottom get app icon added on side menu in web only

### PR DESCRIPTION
**Pull request by mahmed0715**
Added get app icon on bottom on web left menu.

### CODE REVIEW

File: app/components/Menu/index.js [hunk 1]
```diff
@@ -138,6 +138,20 @@ export default class Menu extends Component {
           </div>
         ))}
         {this.props.userProfile ? this.renderShareButtons() : null}
+        <div className="bottom-banner">
+          <a
+            target="_blank"
+            href="https://apps.apple.com/us/app/plant-for-the-planet/id1444740626"
```
Comment (norbertschuler) [inline app/components/Menu/index.js]: https://apps.apple.com/app/plant-for-the-planet/id1444740626 also works.

**Comment by norbertschuler** [comment]
Google Badges: https://play.google.com/intl/en_us/badges/
Apple Badges: https://developer.apple.com/app-store/marketing/guidelines/#section-badges

**Comment by mahmed0715** [comment]
#1293

**Comment by mahmed0715** [comment]
@sagararyal please check localized badge addded

**Comment by mahmed0715** [comment]
@sagararyal the size of the google play icon doesnt match with the apple icon(actually not size, but the google icon has a white border around, that looks smaller than apple icon). Please check/match if we are okay to use that?

**Comment by mahmed0715** [comment]
Screenshot from 2019-09-02 23-12-32

**Comment by sagararyal** [comment]
in that case you can download the icons at: https://play.google.com/intl/en_us/badges/
and resize?

File: app/assets/index.js [hunk 2]
```diff
@@ -138,7 +138,10 @@ import forward from './images/forward.png';
 import successAnimated from './images/successAnimated.gif';
 import trees from './images/trees.png';
 import greencalendar from './images/greencalendar.png';
-
+import googlePlayBadge_en from './images/en/google-play-badge.jpg';
```
Comment (norbertschuler) [inline app/assets/index.js]: Is it only we who think it's weird to use JPG for the Google banner and SVG for the Apple banner? Why not both SVG or both PNG?

Comment (mahmed0715) [reply app/assets/index.js]: The images in svg are not good enough for both, we used svg for IOS from the package, but got google badge from their site and resized/edited to look similar, thus this happened.

**Comment by mahmed0715** [comment]
@norbertschuler @sagararyal please see now. resized image for google to look similar.

File: app/components/Menu/index.js [hunk 3]
```diff
@@ -7,7 +7,10 @@ import { getLocalRoute } from '../../actions/apiRouting';
 import { context } from '../../config';
 import { allowedUrls } from '../../config/socialShare';
 import { FacebookShareButton, TwitterShareButton } from 'react-share';
+import { getLocale } from '../../actions/getLocale';
 
+let userLang = getLocale() || 'en'; // en|de
+userLang = userLang.split('_')[0];
```
Comment (norbertschuler) [inline app/components/Menu/index.js]: These above two lines are not necessary as this is already done within `getLocale()`, isn't it?

File: app/components/Menu/index.js [hunk 4]
```diff
@@ -138,6 +141,20 @@ export default class Menu extends Component {
           </div>
         ))}
         {this.props.userProfile ? this.renderShareButtons() : null}
+        <div className="bottom-banner">
```
Comment (norbertschuler) [inline app/components/Menu/index.js]: If I resize the browser window until the menu does not fit in any more, the banner cover a menu item and if you scroll the menu the banner stay fixed on the menu item they cover before, so you can not select it.

Bildschirmfoto 2019-09-03 um 14 41 16
Bildschirmfoto 2019-09-03 um 14 41 26

Comment (norbertschuler) [reply app/components/Menu/index.js]: On very small devices, e.g. iPhone 5, the menu is shown as responsive layout, but the banner cover the menu again, so some items can not be selected any more.

Bildschirmfoto 2019-09-03 um 14 45 03

Comment (mahmed0715) [reply app/components/Menu/index.js]: we should show just under the end of the menu items?not just fixed position(as we do fixed position like at very bottom) @norbertschuler

Comment (mahmed0715) [reply app/components/Menu/index.js]: @norbertschuler responsive issue solved.

Comment (norbertschuler) [reply app/components/Menu/index.js]: I think fixing the responsive layout this way is good. The other case where you resize the normal browser window is just an edge case probably no one will experience. So I mark this as resolved.

File: app/components/Menu/index.js (hunk 3 above)
Comment (norbertschuler) [reply app/components/Menu/index.js]: I suggest replacing this code with `let userLang = getLocale();`

Comment (mahmed0715) [reply app/components/Menu/index.js]: changed as requested

**Comment by mahmed0715** [comment]
@norbertschuler please see

**Pull request merged**